doc/source/about.rst

```diff
@@ -67,14 +67,15 @@ Here we give a simple example of using an IMPROVER CLI to threshold data, moving
 
 .. code:: console
 
-    bin/improver threshold input_file.nc output_file.nc 5 10 15 20 --threshold_units m/s
+    bin/improver threshold input.nc --threshold-values 5,10,15,20 --threshold-units m/s --output output.nc
 
-
-* input_file.nc is a netCDF file containing a forecast diagnostic data cube, e.g. wind speeds across an x-y grid at a given time.
+* threshold is the name of file to be called. In cases where the name has under-scores these should be written as hyphens e.g. snow_fraction would be written as snow-fraction.
```

Comment:
I think that this would be more accurate:
```suggestion
* threshold is the name of the CLI to be called. In cases where the name of the CLI has underscores these should be written as hyphens e.g. snow_fraction would be written as snow-fraction.
```

Reply:
Done. Thanks for spotting that.